forwards_paper.tex

```diff
@@ -424,59 +434,76 @@ \subsection*{Tree sequence simplification}
+        It has 10 node records (one for each individual)
         and 14 edge records (one for each distinctly inherited segment).
-        Blue numbers denote crossing over locations in each meiosis.
+        Blue numbers denote crossing over locations in each meiosis --
+        for instance, $C$ and $D$ were parents to $F$,
+        who inherited the left 70\% of the chromosome from $D$ and the remainder from $C$.
         The individuals $B$, $C$, and $D$ inherit clonally from $A$.
         \textbf{Center:} the five distinct trees relating all individuals to each other
         found across the chromosome (blue numbers denote locations on the chromosome).
         Labels after simplification are shown in red.
         \textbf{Bottom:} tables recording the tree sequence after simplification
         with nodes $I$ and $J$ as samples.
         The mapping from labels in the forwards time simulation to nodes in the tree sequence
-        is shown in red,
-        which allows additional records to be added as the simulation progresses.
+        is shown in red.
+        % which allows additional records to be added as the simulation progresses.
         \label{fig:method_diagram}
     }
 \end{figure}
 
-In the current context of running forwards in time simulations, the tables of
-nodes and edges that we record all of history for everyone alive at any time through the simulation.
+The tree sequences produced by forwards simulations, 
+record all of history for everyone alive at any time through the simulation.
 This is much more than we need to reconstruct the genealogies and sequences
-of the current day population. We can use the process of simplification here to
-reduce this down to the minimal set of nodes and edges required to represent
-the information that we are interested in. We can also use simplification if  we have some very
+of the current population.
+Simplification is essential to reduce this 
+to a manageable quantity that still contains all
+the information that we are interested in. 
+Simplification is also useful if we have a
 large tree sequence representing a large dataset and we wish to extract the
 information relevant to a subset of the samples.
 
-Roughly, simplification works by tracing ancestry from the samples backwards
-through the recorded history, adding node and edge records to the output
-only when coalescent events are reached. This works exactly as in \msprime,
-allowing substantial re-use of algorithms; the main difference being that
-parental choice, mutations, and recombination locations are determined by the
-input tree sequence rather than randomly generated. An example is shown in
-Figure \ref{fig:method_diagram}.
+The approach that we take is based on Hudson's algorithm for simulating
+the coalescent with recombination~\citep{hudson1983properties,kelleher2016efficient};
+the implementaiton of simplify parallels the implementation of Hudson's algorithm in \msprime.
+Conceptually, this works by 
```

Comment:
does the "color" depiction speak to you?  I think we can put a high level description here ("results") and an actual algorithm in the methods.